gui/modes/CommandPressureMode.qml

```diff
@@ -0,0 +1,308 @@
+import QtQuick 2.11
+import QtQuick.Layouts 1.3
+import QtQuick.Controls 2.4
+
+import "../controls"
+import ".."
+
+Mode {
+    Timer // TODO: Make data sources be updated as we get data
+    {
+        id: refreshTimer
+        interval: 1 / 15 * 1000 // 60 Hz
+        running: true
+        repeat: true
+        onTriggered: {
+            // TODO: guiState as a contextObject is not great
```

Comment:
Could you elaborate on this or file a bug? I added this stuff and I'd love to know what's the right way to do it.

Reply:
It is not a bug, just that when we create context object we lose readability, also might clash with properties of the same name. This presentation gives a good explanation:

https://youtu.be/JbSNobpE_sM?t=638

I can file a bug as you suggest, but I have the impression that this code will change a lot once we start implementing the actual functionality, so this will go away very soon. Let me know if you feel it is necessary to have a separate ticket for this. Otherwise, this can be resolve when looking into the implementation of this mode.